pkg/controller/component/lifecycle/kbagent.go

```diff
@@ -138,13 +229,13 @@ func (a *kbagent) buildActionRequest(ctx context.Context, cli client.Reader, la
 	return req, nil
 }
 
-func (a *kbagent) callActionWithSelector(ctx context.Context, spec *appsv1alpha1.Action, la lifecycleAction, req *proto.ActionRequest) error {
+func (a *kbagent) callActionWithSelector(ctx context.Context, spec *appsv1alpha1.Action, lfa lifecycleAction, req *proto.ActionRequest) error {
 	pods, err := a.selectTargetPods(spec)
```

Comment:
Cannot specify that memberJoin should be executed on the newly added Pod.

Reply:
I'm not sure if such a selector is necessary.

For now, most addon implements memberJoin in new Pod's ENTRYPOINT script.

Reply:
Maybe we should add an option in TargetPodSelector, like "Operated" or "Current"

Reply:
"Current" is the default behavior.

Reply:
Yeah, should we add the "Current" option explicitly in API?